tools/make/packaging.mk

```
@@ -175,7 +176,7 @@ ifeq ($(USE_CONTAINER),1)
 	$(RERUN_IN_CONTAINER)
 else
 	$(call generate_fpm,deb,amd64,amd64,$(PACKAGE_PREFIX).amd64.deb)
-	$(call generate_fpm,rpm,x86_64,amd64,$(PACKAGE_PREFIX).amd64.rpm)
+	$(call generate_fpm,rpm,x86_64,amd64,$(PACKAGE_PREFIX).amd64.rpm,--rpm-rpmbuild-define "_build_id_links none")
```

See suggestion above; if that change is made, this is no longer necessary (plus, this is only doing it for AMD64 RPMs, when we probably want this change for all of them).

```suggestion
	$(call generate_fpm,rpm,x86_64,amd64,$(PACKAGE_PREFIX).amd64.rpm)
```